tests/unit/amo/components/Errors/TestAuthExpired.js

```diff
@@ -13,15 +13,15 @@ import {
   fakeI18n,
   shallowUntilTarget,
 } from 'tests/unit/helpers';
-import { API_ERROR_SIGNATURE_EXPIRED } from 'amo/constants';
+import { API_ERRORS_SESSION_EXPIRY } from 'amo/constants';
```

Comment:
I don't think we need to update this test like this, because this component doesn't actually care *what* the error code is. It's the components that call this that look at an error code to decide whether to display it or not, so just testing this with any error code is actually sufficient.